### PR TITLE
Delete openDesktop.xml

### PR DESCRIPTION
**Pull request by Bisaloo**
That ruleset has been bothering me for a long time. Most domains are broken or for sale. The rest just are mainly redirects.

I will maybe add the few domains that are working in separate rulesets.

### CODE REVIEW

**Comment by Bisaloo** [comment]
In another PR (because this seems unrelated to openDesktop):

- Update [`VideoLAN.xml`](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/VideoLAN.xml) to include https://addons.videolan.org
- Create a ruleset for xfce-help.org

**Comment by J0WI** [comment]
Can we split this in multiple PRs?

**Comment by Bisaloo** [comment]
@J0WI, I am not sure how to split it while making sense.

I know it may seem daunting due to the number of files and commits but I think it is a pretty straightforward PR:

1. [I removed all subdomains other than `opendesktop.org` from the open desktop ruleset](https://github.com/EFForg/https-everywhere/pull/10190/files#diff-f7b8c735e627741c6570a6101ab423e1).
2. I created a trivial ruleset for each one of the related domain (only `^`, `www` and a trivial rule).

File: src/chrome/content/rules/openDesktop-mismatches.xml [hunk 1]
```diff
@@ -1,150 +0,0 @@
...
-	<target host="www.box-look.org" />
-
-	<target host="cli-apps.org" />
-	<target host="www.cli-apps.org" />
-
-	<target host="debian-art.org" />
-	<target host="www.debian-art.org" />
-
-	<target host="e17-stuff.org" />
-	<target host="www.e17-stuff.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: WFM

File: src/chrome/content/rules/openDesktop-mismatches.xml [hunk 2]
```diff
@@ -1,150 +0,0 @@
...
-	<target host="www.gnomefiles.org" />
-
-	<target host="gnome-help.org" />
-	<target host="www.gnome-help.org" />
-
-	<target host="gnome-look.org" />
-	<target host="www.gnome-look.org" />
-
-	<target host="gtk-apps.org" />
-	<target host="www.gtk-apps.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: WFM

File: src/chrome/content/rules/openDesktop-mismatches.xml [hunk 3]
```diff
@@ -1,150 +0,0 @@
...
-
-	<target host="java-apps.org" />
-	<target host="www.java-apps.org" />
-
-	<target host="kde-apps.org" />
-	<target host="www.kde-apps.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: WFM

File: src/chrome/content/rules/openDesktop-mismatches.xml [hunk 4]
```diff
@@ -1,150 +0,0 @@
...
-
-	<target host="kde-files.org" />
-	<target host="www.kde-files.org" />
-
-	<target host="kde-help.org" />
-	<target host="www.kde-help.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: WFM

File: src/chrome/content/rules/openDesktop-mismatches.xml [hunk 5]
```diff
@@ -1,150 +0,0 @@
...
-
-	<target host="kde-look.org" />
-	<target host="www.kde-look.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: WFM

File: src/chrome/content/rules/openDesktop-mismatches.xml [hunk 6]
```diff
@@ -1,150 +0,0 @@
...
-	<target host="www.open-pc.com" />
-
-	<target host="openskillz.com" />
-	<target host="www.openskillz.com" />
-
-	<target host="opentemplate.org" />
-	<target host="www.opentemplate.org" />
-
-	<target host="qt-apps.org" />
-	<target host="www.qt-apps.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: WFM

File: src/chrome/content/rules/openDesktop-mismatches.xml [hunk 7]
```diff
@@ -1,150 +0,0 @@
...
-	<target host="scribusstuff.org" />
-	<target host="www.scribusstuff.org" />
-
-	<target host="suse-art.org" />
-	<target host="www.suse-art.org" />
-
-	<target host="ubuntu-art.org" />
-	<target host="www.ubuntu-art.org" />
-
-	<target host="addons.videolan.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: WFM

File: src/chrome/content/rules/openDesktop-mismatches.xml [hunk 8]
```diff
@@ -1,150 +0,0 @@
...
-
-	<target host="vlc-addons.org" />
-	<target host="www.vlc-addons.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: WFM

File: src/chrome/content/rules/openDesktop-mismatches.xml [hunk 9]
```diff
@@ -1,150 +0,0 @@
...
-
-	<target host="wine-apps.org" />
-	<target host="www.wine-apps.org" />
-
-	<target host="xfce-help.org" />
-	<target host="www.xfce-help.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: WFM

**Comment by J0WI** [comment]
One part to remove all inactive/broken targets and anotherpart to split the ruleset would have been easier to review.

File: src/chrome/content/rules/openDesktop-mismatches.xml (hunk 4 above)
Comment (Bisaloo) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: This domain is still under construction. It's probably safer to wait before adding it.

File: src/chrome/content/rules/openDesktop-mismatches.xml (hunk 7 above)
Comment (Bisaloo) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: https://github.com/EFForg/https-everywhere/pull/10190#issuecomment-306440588

Comment (Bisaloo) [inline src/chrome/content/rules/openDesktop-mismatches.xml]: https://github.com/EFForg/https-everywhere/pull/11083

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**